experimental/dds/tree2/src/feature-libraries/forestSummarizer.ts

```diff
@@ -96,12 +117,15 @@ export class ForestSummarizer implements Summarizable {
 			const treeBufferString = bufferToString(treeBuffer, "utf8");
 			// TODO: this code is parsing data without an optional validator, this should be defined in a typebox schema as part of the
 			// forest summary format.
-			const fields = parse(treeBufferString) as [FieldKey, JsonableTree[]][];
-
+			const fields = parse(treeBufferString) as [FieldKey, EncodedChunk][];
 			const delta: [FieldKey, Delta.Insert[]][] = fields.map(([fieldKey, content]) => {
+				const jsonableTree = mapCursorField(
```

Comment:
this conversion is unneeded. Since you just need a cursor, `decode(content).cursor()` can be used as the Delta's content below, and you don't need to generate a jsonable tree then make a cursor from that.

Reply:
I have tried to prevent generating a jsonable tree to convert back to a cursor by changing to the following (shown below), but for some reason this does not convert the field cursor into an array of node cursors and the resulting cursors are still in `fields` mode (which leads to an error).

`const cursors = mapCursorField(
	decode(content).cursor(),
	(cursor) => cursor as ITreeCursorSynchronous
)`

However, if I convert to a jsonable tree first (shown below), the cursors seem to be properly converted and it works.

`const cursors = mapCursorField(
	decode(content).cursor(),
	jsonableTreeFromCursor,
).map(singleTextCursor);`

Reply:
updated to `const cursors = mapCursorField(decode(content).cursor(), (cursor) => cursor.fork());`